src/javascript/app/pages/user/metatrader/metatrader.js

```diff
@@ -22,7 +22,12 @@ const MetaTrader = (() => {
         BinarySocket.wait('landing_company', 'get_account_status', 'statement').then(async () => {
             if (isEligible()) {
                 if (Client.get('is_virtual')) {
-                    await addAllAccounts();
+                    try {
+                        await addAllAccounts();
+                    } catch (error) {
+                        // eslint-disable-next-line no-console
+                        console.warn(error.message);
```

Comment:
why are we just warning in console?

Reply:
I wanted to ignore it, but I thought I left a trace for further debugging. I can just do nothing in the error block, what do you suggest?